packages/nodejs/package.json

```diff
@@ -60,7 +60,7 @@
         "build-dev": "tsc"
     },
     "dependencies": {
-        "@concordium/common-sdk": "6.4.1",
+        "@concordium/common-sdk": "6.4.2",
```

Comment:
Remember to update the lock file (just run `yarn` to do so) when updating dependencies :upside_down_face: